### ISSUE TITLE
alias boot fails

### ISSUE THREAD

**Issue by eed3si9n** [sbt/sbt#4869]
## steps
sbt version: 1.3.0-bin-20190713T235600

Create `.sbtrc`

```
alias boot = ;reload ;project root ;iflast shell
```

run sbt.

## problem

```
[error] boot usage:
[error]
[error]
[error] boot
[error]     ^
```

## expectation

sbt boots up fine.

## notes

https://github.com/sbt/sbt/pull/4811

**Comment by eatkins** [comment]
Yes. I just noticed this myself.

**Comment by eatkins** [comment]
The problem is that they are overriding the boot command. I'm not sure yet why the post 1.3.0-RC2 multi parser changes break it. If the alias were named something different, it would work.

**Comment by eatkins** [comment]
I don't know about this. I'm not convinced we should be supporting aliasing boot. I changed the aliasrc scripted test to have a line: `alias boot = clean`. It fails with sbt 1.2.8.

**Comment by eatkins** [comment]
I should add that it fails in the exact same way as the initial problem description above.

**Comment by eed3si9n** [comment]
> I don't know about this. I'm not convinced we should be supporting aliasing boot. 

I don't use it myself, but I feel like it's a feature that's advertised for a very long time.

**Comment by eed3si9n** [comment]
2011 by Mark - https://groups.google.com/d/msg/simple-build-tool/oe_3JVtIYB4/XWRZHYRTLA0J

> For different startup behavior, 'boot' can be overridden with an alias in .sbtrc.  So, you could redefine boot to be:

**Comment by eatkins** [comment]
Right. But the thing is that we can't even write a scripted test to guard against this feature regressing since at least 1.2.8. This is Mark getting too cute in my opinion. I'll continue looking for a workaround but this is a very frustrating feature implementation.

**Comment by eed3si9n** [comment]
A direction I think we should consider is accept #4808 as a corner case collateral damage, and revert #4811. You mentioned that there were already situation that it won't work like command definition  `;` followed by the execution of the command.

**Comment by eatkins** [comment]
Right, so the problem here is that the `project` command doesn't exist until after `reload`. If you change it to `alias boot = ;reload; iflast shell` then things work. As a side note, this alias is completely pointless in the kittens project. It does the exact same thing as the default boot.

**Comment by eed3si9n** [comment]
Fixed in #4871
